skfda/preprocessing/dim_reduction/feature_extraction/_fda_feature_union.py

```diff
@@ -0,0 +1,115 @@
+
+
+class FdaFeatureUnion(FeatureUnion):
+    """Concatenates results of multiple functional transformer objects.
+
+    This estimator applies a list of transformer objects in parallel to the
+    input data, then concatenates the results (They can be either FDataGrid
+    and FDataBasis objects or multivariate data itself).This is useful to
+    combine several feature extraction mechanisms into a single transformer.
+    Parameters of the transformers may be set using its name and the parameter
+    name separated by a '__'. A transformer may be replaced entirely by
+    setting the parameter with its name to another transformer,
+    or removed by setting to 'drop'.
+
+    Parameters:
+        transformer_list: list of tuple
+            List of tuple containing `(str, transformer)`. The first element
+            of the tuple is name affected to the transformer while the
+            second element is a scikit-learn transformer instance.
+            The transformer instance can also be `"drop"` for it to be
+            ignored.
+        n_jobs: int
+            Number of jobs to run in parallel.
+            ``None`` means 1 unless in a :obj:`joblib.parallel_backend`
+            context.
+            ``-1`` means using all processors.
+            The default value is None
+        transformer_weights: dict
+            Multiplicative weights for features per transformer.
+            Keys are transformer names, values the weights.
+            Raises ValueError if key not present in ``transformer_list``.
+        verbose: bool
+            If True, the time elapsed while fitting each transformer will be
+            printed as it is completed. By default the value is False
+        array_output: bool
+            indicates if the transformed data is requested to be a NumPy array
+            output. By default the value is False.
+
+    Examples:
+    Firstly we will import the Berkeley Growth Study data set
+    >>> from skfda.datasets import fetch_growth
+    >>> X,y = fetch_growth(return_X_y=True)
+
+    Then we need to import the transformers we want to use. In our case we
+    will use Minimum Redundancy Maximum Relevance.
+    Evaluation Transformer returns the original curve, and as it is helpful,
+    we will concatenate it to the already metioned transformer.
+    >>> from skfda.preprocessing.dim_reduction.feature_extraction import (
+    ...     FdaFeatureUnion,
+    ... )
+    >>> from skfda.preprocessing.dim_reduction.variable_selection import (
+    ...     MinimumRedundancyMaximumRelevance,
+    ... )
+    >>> from skfda.representation import EvaluationTransformer
+    >>> import numpy as np
+
+    Finally we apply fit and transform.
+    >>> union = FdaFeatureUnion(
+    ...     [
+    ...        ("mrmr", MinimumRedundancyMaximumRelevance()),
+    ...        ("eval", EvaluationTransformer()),
+    ...     ],
+    ...     array_output=True,
+    ... )
+    >>> np.around(union.fit_transform(X,y), decimals = 2)
```

Comment:
You shouldn't use whitespace around keyword parameters equals:
```suggestion
    >>> np.around(union.fit_transform(X,y), decimals=2)
```